lib/usd/ui/USDImportDialog.h

```diff
@@ -79,6 +86,8 @@ private Q_SLOTS:
 	std::unique_ptr<TreeModel> fTreeModel;
 	// Reference to the Proxy Model used to sort and filter the USD file hierarchy:
 	std::unique_ptr<QSortFilterProxyModel> fProxyModel;
+	// Reference to the delegate we set on the tree view:
+	std::unique_ptr<ItemDelegate> fItemDelegate;
```

Comment:
Moving the delegate to a member variable fixes the memory leak and makes it easier to hook one of its signals up to the model.